CPackLists.txt

```diff
@@ -64,27 +67,33 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
    message( STATUS "make package" )
 ENDIF()
 
-
-message( STATUS "\nOption to install using 'sudo make install" )
+IF(NOT MINGW)
+   message( STATUS "\nOption to install using 'sudo make install'" )
+ELSE()
```

Comment:
We can simplify this to just say the message you put in. I.e no `else` or `if` case needed

```
message( STATUS "\nOption to install using 'make install'" )
```